certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+{
+  "x-generator": "NSwag v13.14.8.0 (NJsonSchema v10.6.1.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "AIForged",
+    "description": "Integration with AI Forged system",
+    "termsOfService": "None",
+    "contact": {
+      "name": "AIForged",
+      "url": "https://www.aiforged.com",
+      "email": "support@aiforged.com"
+    },
+    "license": {
+      "name": "Proprietary license"
+    },
+    "version": "v0.9"
```

Comment:
@aiforged Since it is a new version for your connector, it should start with "1.0" or "1.0.0".

Reply:
sure - the paconn did not complain

Reply:
ok, we updated version to 1.0 and other changes will be pushed soon